src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../types';
+import { findPersonByName } from '../utils/findPersonByName';
+
+interface Props {
+  person: Person;
+  people: Person[];
+}
+
+export const PersonInfo: FC<Props> = ({ person, people }) => {
```

Comment:
Consider using destructuring for the `useParams` hook. This will make your code a bit cleaner. For example: `const { personSlug } = useParams<{ personSlug: string }>();`